collectors/python.d.plugin/openldap/openldap.conf

```diff
@@ -0,0 +1,74 @@
+#
+# name: value
+#
+# There are 2 sections:
+#  - global variables
+#  - one or more JOBS
+#
+# JOBS allow you to collect values from multiple sources.
+# Each source will have its own set of charts.
+#
+# JOB parameters have to be indented (using spaces only, example below).
+
+# ----------------------------------------------------------------------
+# Global Variables
+# These variables set the defaults for all JOBs, however each JOB
+# may define its own, overriding the defaults.
+
+# update_every sets the default data collection frequency.
+# If unset, the python.d.plugin default is used.
+# postfix is slow, so once every 10 seconds
+update_every: 10
+
+# priority controls the order of charts at the netdata dashboard.
+# Lower numbers move the charts towards the top of the page.
+# If unset, the default for python.d.plugin is used.
+# priority: 60000
+
+# retries sets the number of retries to be made in case of failures.
+# If unset, the default for python.d.plugin is used.
+# Attempts to restore the service are made once every update_every
+# and only if the module has collected values in the past.
+# retries: 60
+
+# autodetection_retry sets the job re-check interval in seconds.
+# The job is not deleted if check fails.
+# Attempts to start the job are made once every autodetection_retry.
+# This feature is disabled by default.
+# autodetection_retry: 0
+
+# ----------------------------------------------------------------------
+# JOBS (data collection sources)
+#
+# The default JOBS share the same *name*. JOBS with the same name
+# are mutually exclusive. Only one of them will be allowed running at
+# any time. This allows autodetection to try several alternatives and
+# pick the one that works.
+#
+# Any number of jobs is supported.
+#
+# All python.d.plugin JOBS (for all its modules) support a set of
+# predefined parameters. These are:
+#
+# job_name:
+#     name: myname            # the JOB's name as it will appear at the
+#                             # dashboard (by default is the job_name)
+#                             # JOBs sharing a name are mutually exclusive
+#     update_every: 1         # the JOB's data collection frequency
+#     priority: 60000         # the JOB's order on the dashboard
+#     retries: 60             # the JOB's number of restoration attempts
+#     autodetection_retry: 0  # the JOB's re-check interval in seconds
+#
+# ----------------------------------------------------------------------
+# OPENLDAP EXTRA PARAMETERS
+
+# Set here your LDAP connection settings
+
+username : "cn=admin,dc=example,dc=com"   # The bind user with right to access monitor statistics
+password : "yourpass"                     # The password for the binded user
+#server   : 'localhost'                    # The listening address of the LDAP server
+#port     : 389                            # The listening port of the LDAP server
+#timeout  : 1                              # Seconds to timeout if no connection exists
```

Comment:
please comment out all extra parameters (`username` and `password` too)

Reply:
ok. So finaly wilI I change the code to allow anonymous?

Reply:
If you want to deny anonymous connection you should add

```python
if not (self.user and self.password):
    self.error('anonymous connections are not allowed, please specify user and password')
    return False
```

right after `if not HAS_LDAP: ...`  check

I am in doubt about that. Since it's technically allowed and `ldap` package allows it i think we should allow it too.

@Ferroin you decide!

Reply:
As a general rule I would say no, but we really aren't in the business of telling people how they should secure their systems, so I think supporting it is probably fine.

Reply:
Finally I aggree with @Ferroin 😀  But now maybe I will need one more day because I am out of time now 😞.

Reply:
You need to add one more think and we are good - https://github.com/netdata/netdata/pull/4513#discussion_r233155374
!